ui/src/actions/harvester.js

```diff
@@ -68,22 +68,22 @@ export function harvestAndLoadCrystal(xtalUUID, successCb = null) {
       }
     } catch (error) {
       dispatch(showErrorPanel(true, error.response.headers.get('message')));
-      throw new Error('Server refused to Harveste or Load Crystal');
+      throw new Error('Server refused to harvest or load crystal');
     }
   };
 }
 
-export function sendDataCollectionToCrims() {
+export function sendDCToCrims() {
```

Comment:
```suggestion
export function sendDcToCrims() {
```

I prefer like this, this seems more correct to me (and it hurts my eyes less). Also because we would not write `send_d_c_to_crims`.

But then above we have `sendDataCollectionInfoToCrims`. So we could also go with:

```suggestion
  sendDataCollectionToCrims,
```